### PR TITLE
Update AzureFirewallWorkbook.json

### PR DESCRIPTION
**Pull request by chboeh**
Fixes #

## Proposed Changes

  - Latest update of the Azure Firewall Workbook ( Been tested for 3 months )
  -
  -

### CODE REVIEW

File: Workbooks/AzureFirewallWorkbook.json [hunk 1]
```diff
@@ -1624,11 +2725,69 @@
...
+        "value": "AFInvestigate"
+      },
+      "name": "query - 58"
     }
   ],
   "fallbackResourceIds": [
-    "/subscriptions/7b76bfbc-cb1e-4df1-b6e8-b826eef6c592/resourcegroups/soc/providers/microsoft.operationalinsights/workspaces/cybersecuritysoc"
+    "Azure Monitor"
   ],
-  "fromTemplateId": "sentinel-AzureFirewall",
```
Comment (v-jayakal) [inline Workbooks/AzureFirewallWorkbook.json:1632]: Should be 'sentinel-AzureFirewall', specific to this workbook, why this value has been changed? Can you change that back?

File: Workbooks/AzureFirewallWorkbook.json [hunk 2]
```diff
@@ -1624,11 +2725,69 @@
...
+      "conditionalVisibility": {
+        "parameterName": "selectedTab",
+        "comparison": "isEqualTo",
+        "value": "AFInvestigate"
+      },
+      "name": "query - 58"
     }
   ],
   "fallbackResourceIds": [
-    "/subscriptions/7b76bfbc-cb1e-4df1-b6e8-b826eef6c592/resourcegroups/soc/providers/microsoft.operationalinsights/workspaces/cybersecuritysoc"
```
Comment (v-jayakal) [inline Workbooks/AzureFirewallWorkbook.json:1630]: Should not have the hard coded id (subscriptions/resource group), this is the reason for the validation failure as well.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

**Comment by v-jayakal** [comment]
@chboeh - Can you please update the version of the AzureFirewallWorkbook in workbook metadata? Thanks.

File: Workbooks/AzureFirewallWorkbook.json (hunk 2 above)
Comment (chboeh) [reply Workbooks/AzureFirewallWorkbook.json:1630]: This was on the old template, that's currently being used today. The new update doesn't have it.. are you wanting it added back into the shared template? @v-jayakal

File: Workbooks/AzureFirewallWorkbook.json (hunk 1 above)
Comment (chboeh) [reply Workbooks/AzureFirewallWorkbook.json:1632]: Good catch, corrected and updated. @v-jayakal

**Comment by chboeh** [comment]
> @chboeh - Can you please update the version of the AzureFirewallWorkbook in workbook metadata? Thanks.

Done.

File: Workbooks/AzureFirewallWorkbook.json [hunk 3]
```diff
@@ -187,14 +181,265 @@
...
+            "jsonData": "[\r\n { \"value\": \"Yes\", \"label\": \"Yes\", \"selected\":true},\r\n {\"value\": \"No\", \"label\": \"No\" }\r\n]",
+            "value": "No"
+          },
+          {
+            "id": "53aa8daf-29de-40a1-9147-c0970e32fac4",
+            "version": "KqlParameterItem/1.0",
+            "name": "GeoLocation",
+            "label": "GeoLocationCSV",
+            "type": 2,
+            "description": "Supplied as a value for GeoLocation lookup on queries: Read more here - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/ipv4-lookup-plugin#ipv4-lookup---matching-rows-only",
```
Comment (v-rucdu) [inline Workbooks/AzureFirewallWorkbook.json:218]: Can you please remove the locale from the documentation link here and all its occurrences in this file? This is the reason for Validation failure.

Use this link : **https://docs.microsoft.com/azure/data-explorer/kusto/query/ipv4-lookup-plugin#ipv4-lookup---matching-rows-only**

File: Workbooks/AzureFirewallWorkbook.json (hunk 2 above)
Comment (v-rucdu) [reply Workbooks/AzureFirewallWorkbook.json:1630]: @chboeh: Jaya wants to say is that, please remove the content from fallbackResourceIds attribute and make it  as 
**"fallbackResourceIds": []**

**Review by v-rucdu — CHANGES_REQUESTED** [review]

**Comment by chboeh** [comment]
Please clarify @v-rucdu, i don't see this in the template.

image

File: Workbooks/AzureFirewallWorkbook.json (hunk 3 above)
Comment (chboeh) [reply Workbooks/AzureFirewallWorkbook.json]: @v-rucdu,

Done.

**Review by v-rucdu — APPROVED** [review]

**Comment by v-rucdu** [comment]
> Please clarify @v-rucdu, i don't see this in the template.
> 
> image


@chboeh : Yes, it should not contain subscription/resource ids rest anything is ok. No need to change anything here.

Thanks!!!

**Pull request merged**